.github/workflows/bats.yaml

```diff
@@ -25,11 +25,9 @@ on:
         type: string
       kubernetes-version:
         description: Primary Kubernetes version to test
-        default: '1.22.7'
```

Comment:
I know it makes maintenance a little harder, but I would prefer to keep the default value here as well, so the user can see what would be used by default.

Maybe add a comment that the value also needs to be changed later in the workflow if the default is updated here?

I could not find a way to specify the value only once, and still have it displayed in the dialog.